packages/react/src/River/River.module.css

```diff
@@ -2,7 +2,7 @@
   display: grid;
   gap: var(--brand-River-gap-column-medium);
   grid-auto-rows: min-content;
-  margin: var(--brand-River-layout-margin-vertical);
+  padding: var(--brand-River-layout-margin-vertical);
```

Comment:
one for later, but can we should rename this token to something generic like spacing, or more explicit like padding.

Reply:
going further: if we end up using a container for all River items, this could be renamed as `--brand-River-gap-vertical` or simply `--brand-River-gap`  and use it this way:

 ```css 
gap: var(--brand-River-gap);
```